flow_screen_components/richTextInputFSC/README.md

```diff
@@ -1,3 +1,45 @@
 # Rich Text Edit Control #
 
 https://unofficialsf.com/rich-text-input-flow-screen-component/
+
+
+# inputRichTextFSC
+Rich Text enhancements for Flows
+
+### Summary:
+IMPORTANT:  Use richerText = true input attribute if you want to leverage these enhanced features.  Otherwise leave empty and regular input Rich Text component will be used.
+1)	Block disallowed words and symbols
+    a.	Control whether this is a block that prevents next/finish of flow.
+2)	Search and Replace custom text
+3)	Auto-Replace suggested words
+4)	Revert mass changes from search/replace or suggested words
+
+### Admin Instructions:
```

Comment:
Group your instructions into functional categories that start with the functionality:

**Enabling and Disabling Features**
[   ]

**Replacing Text**
Specify replacement text using a JSON map. Example: {"Test": "GreatTest™"}
When does replacement happen? 


**Disallowed Words and Symbols**
Specify words as a comma-separated list....
By default, the component prevents Flow from transitioning if a disallowed word or symbol is present. If you want to only warn, but still allow transition, set warnOnly to True

Reply:
Great, thanks.  This will be in next update.